addon/mixins/taggable-mixin.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Mixin.create({
             Ember.A(currentTags);
```

Comment:
Hmm. It appears that `Ember.A` really does work this way- weird, but interesting! Thanks.

Page suggesting it modifies the original in place: https://guides.emberjs.com/v2.3.0/configuring-ember/disabling-prototype-extensions/#toc_arrays

Page suggesting that it totally doesn't modify the original object:
http://emberjs.com/api/#method_A